llvm/tools/llvm-lto/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ set(LLVM_LINK_COMPONENTS
   Support
   Target
   TargetParser
+  CAS
+  RemoteCachingService
+  RemoteNullService
```

Comment:
Just for my understanding: Why does this depend on CAS but not on MCCAS?

Reply:
This is reasonable. We only have full object caching in ThinLTO.